src/main/java/uk/gov/hmcts/reform/civil/service/DeadlinesCalculator.java

```diff
@@ -60,6 +60,14 @@ public LocalDate calculateFirstWorkingDay(LocalDate date) {
         return date;
     }
 
+    public LocalDate plusWorkingDays(LocalDate date, int workingDays) {
```

Comment:
youve missed the tests for this class

Reply:
also i dont think this works as the `getNextWorkingDay` method looks like it only returns the next working day if the current day is not a working day. 
so if its a monday, line 66 will return the same monday so it doesnt look like it will progress